app/validators/form_finance_validator.rb

```diff
@@ -0,0 +1,61 @@
+module FormFinanceValidator
+  extend ActiveSupport::Concern
+
+  VALID_MINIMUM_LENGTH = '8'.freeze
```

Comment:
for some reason I had it in my head that constants must be strings (or at least cannot be integers). This seems to be a mistaken idea. Updated at 4c347d3